doc/jointcal.tex

```diff
@@ -29,527 +43,603 @@
+
+The fit consists of minimizing, for photometry, 
+\begin{align}
+\chi^{2} & =\sum_{\gamma,i}[M_{\gamma}(S_{\gamma,i})-F_{i}]^{T}W_{\gamma,i}[M_{\gamma}(S_{\gamma,i})-F_{i}] & \textrm{(meas. terms)}\nonumber \\
+ & +\sum_{j}[F_{j}-R_{j}]^{T}W_{j}[F_{j}-R_{j}] & \textrm{(ref. terms)}\label{eq:photometry-chi2}
+\end{align}
+and for astrometry, taking into account the projection from the sky
+to the tangent plane $P$:
 \begin{align}
-\chi^2 & =  \sum_{\gamma,i}  [M_\gamma(X_{\gamma,i})-P_{\gamma}(F_k) ]^T W_{\gamma,i}  [M_\gamma(X_{\gamma,i})-P_{\gamma}(F_k) ] & \textrm{(meas. terms)} \nonumber \\
-         &+\sum_j [P(F_j)-P(R_j)]^T W_j [P(F_j)-P(R_j)] & \textrm{(ref. terms)} \label{eq:chi2}
+\chi^{2} & =\sum_{\gamma,i}[M_{\gamma}(S_{\gamma,i})-P_{\gamma}(F_{i})]^{T}W_{\gamma,i}[M_{\gamma}(S_{\gamma,i})-P_{\gamma}(F_{i})] & \textrm{(meas. terms)}\nonumber \\
+ & +\sum_{j}[P(F_{j})-P(R_{j})]^{T}W_{j}[P(F_{j})-P(R_{j})] & \textrm{(ref. terms)}\label{eq:astrometry-chi2}
 \end{align}
-where the first line iterates on all MeasuredStar ($i$) from all
-CcdImage (indexed by $\gamma$), and the second iterates on all RefStar
-($j$). In the first terms, the object at position $F_k$ is the one
-that was measured at position $X_{\gamma,i}$ in image $\gamma$. 
-The association between measurements and objects is 
-described in the above paragraph. 
-
-    The measurement terms compare the measurement positions
-to objects positions, the reference terms compare object positions 
-to reference positions. We need these two sets of terms
-because not all objects $F_k$ in the first terms appear
-in the second terms: there are plenty of objects in the images
-which are not in the reference catalogs, but which constrain the
-mappings $M_\gamma$ to transform measured coordinates to the
-same positions. 
-
-For the measurement terms (first line), the notations are:
+where the first line iterates on all MeasuredStar $\gamma,i$, from
+all CcdImage $\gamma$, and the second iterates on all RefStar $j$.
+In the first terms, the object at position $F_{i}$ is the one that
+was measured at position $S_{\gamma,i}$ in image $\gamma$. The association
+between MeasuredStars, FittedStars, and RefStars is described in Section
+\ref{sec:catalog-association}.
+
+The measurement terms compare the measurement positions/fluxes to
+objects positions/fluxes (the relative astrometry/photometry), the
+reference terms compare object positions/fluxes to reference positions/fluxes
+(the absolute astrometry/photometry). We need these two sets of terms
+because not all objects $F_{i}$ in the first terms appear in the
+second terms: many objects in the images will not be in the reference
+catalogs, but those objects do help to constrain the mappings $M_{\gamma}$.
+In addition, with so many measurements of our sources, we may have
+better overall errors on the positions and fluxes than the reference
+catalog can provide.
+
+In addition to the terms defined in \ref{subsec:Definitions}, eq.
```

Comment:
I think it might be clearer to include these definitions with the others in the previous section.